3-async-await/solution.js

```diff
@@ -20,16 +20,83 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
-    // YOUR SOLUTION GOES HERE
+const { Console } = require("console");
 
-    // You generate your id value here
+async function solution() {
```

Comment:
good job here! 👍 . Just keep on mind recommendations regarding requieres, declare eavery varible with its own const and blanck spaces

Reply:
maybe we cooud declare all the variables at the begning